moveit_core/robot_model/src/floating_joint_model.cpp

```diff
@@ -45,7 +45,7 @@ namespace moveit
 {
 namespace core
 {
-rclcpp::Logger LOGGER = rclcpp::get_logger("robot_model");
+rclcpp::Logger LOGGER_FLOATING_JOINT_MODEL = rclcpp::get_logger("floating_joint_model");
```

Comment:
This makes for such a long namespace, which will cause the INFO/WARN/etc messages to be more multi-line. Is it needed? I've always preferred we keep the syntax terse with just LOGGER

Reply:
In MoveIt1 we had it nice and short with LOGNAME, similarly. Everyone agreed on that standard. We were doing that for MoveIt2 also, but then one of the Acutronic engineers decided he liked the more verbose approach on his own, but i think most of us never wanted it.

Reply:
We replicated already merged PRs but realized that there already are several different solutions to this merged. The long names prevent name conflicts, but alternatively we could switch to anonymous namespaces.